lib/runtime/completions.js

```diff
@@ -77,6 +77,9 @@ class AutoCompleteProvider {
     completion.replacementPrefix = completion._prefix ? completion._prefix : prefix
     const icon = this.makeIcon(completion.icon)
     if (icon) completion.iconHTML = icon
+    // the workaround for https://github.com/atom/autocomplete-plus/issues/868, but can appear verbose
+    if (!completion.description && completion.descriptionMoreURL)
+      completion.description = 'Click the link below to show docs:'
```

Comment:
One thing annoying around adding this feature is this:
without `description`, `descriptionMoreURL` won't work, and the workaround can appear verbose IMO.

Reply:
this means:
![image](https://user-images.githubusercontent.com/40514306/64124418-f39d0380-cde1-11e9-9f2f-fb7a7929f9e3.png)

If you feel annoying this, I can close this PR.
Can I hear your opinion, @pfitzseb ?

Reply:
Would
```
       completion.description = ' '
```
look too weird?

Reply:
Yeah, autocomplete-plus package inserts an newline after `description` anyways...

Reply:
Then let's just fix that with 
```
diff --git a/styles/ink.less b/styles/ink.less
index fa65ab5..e5ab2f8 100644
--- a/styles/ink.less
+++ b/styles/ink.less
@@ -8,8 +8,13 @@
 @import "syntax-variables";
 
 autocomplete-suggestion-list.select-list.popover-list {
-  .suggestion-description-content {
-    overflow-y: hidden;
+  .suggestion-description {
+    line-height: 1;
+    .suggestion-description-content {
+      overflow-y: hidden;
+      display: inline;
+      white-space: normal;
+    }
   }
 }
```
![image](https://user-images.githubusercontent.com/6735977/64127164-dc91ed80-cdb0-11e9-8c3b-9011d2b1f78e.png)
![image](https://user-images.githubusercontent.com/6735977/64127182-e9164600-cdb0-11e9-9da1-1d1f44f17186.png)

Reply:
Looks really good. Thanks for the fix :)